test/swagger.spec.js

```diff
@@ -58,7 +69,7 @@ describe('Swagger docs', function () {
                 require('inject-then'),
                 require('inert'),
                 require('vision'),
-                {register: require('hapi-swagger'), options: {apiVersion: require('../package.json').version}}
+                { register: HapiSwagger, options: swaggerOptions }
             ], () => {
                 server.start(() => {
                     _.forEach(schemas, function (schema) {
```

Comment:
I like it. I'll make the change. After all any improvement to general code quality is worth doing.